File: doc/src_tutorial/xtra_mpi.f90
program main_mpi
	include 'mpif.h'
	call MPI_Init(ierr )
	call MPI_Comm_rank( MPI_COMM_WORLD, myid, ierr )
	call MPI_Comm_size( MPI_COMM_WORLD, ncpus, ierr )
	call MPI_Finalize( ierr )
	stop
end

File: doc/src_tutorial/xtra_mpi.cpp
#include <mpi.h>
int main (int argc, char *argv[])
{
	int my_id, npes;
	MPI_Init(&argc, &argv);
	MPI_Comm_rank(MPI_COMM_WORLD, &my_id);
	MPI_Comm_size(MPI_COMM_WORLD, &npes);
	MPI_Finalize();
	return 0;
}

